include/event_manager.rst

```diff
@@ -109,7 +113,7 @@ Source file
 
 The source file must include the header file for the new event type.
 Define the event type with the :c:macro:`EVENT_TYPE_DEFINE` macro, passing the name of the event type as declared in the header and additional parameters.
-Optionally, you can provide a function that fills buffer with a string version of the event data (used for logging).
+For example, you can provide a function that fills a buffer with a string version of the event data (used for logging).
```

Comment:
I think optionally is better here (you may provide the pointer to function if you want to log event data or put NULL here if you dont want to do it). You can just provide this function or not, you cannot put other function here.

Reply:
It's an example for the "additional parameters" mentioned in the previous sentence. We can be more specific there, but then we need to mention all parameters, not only this one.

Reply:
ok